### PR TITLE
Update issues-workflow.md

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- Remove the mention of `WIP` label. It's not done usually, and we use Zenhub for tracking progress.
- Adapt sprint milestone pattern to match reality
- Adapt backlog milestone to match reality

**Related issue(s)**

### CODE REVIEW

File: issues-workflow.md [hunk 1]
```diff
@@ -57,15 +57,15 @@ The Kyma backlog contains issues that went through the triage, are not closed, a
...
 
 ### Team sprints
-Team Sprint is modeled as a GitHub milestone named with the following pattern: `{TEAM_NAME} Sprint {NUMBER}`. During the planning, a team selects issues from the backlog considering:
+Team Sprint is modeled as a GitHub milestone named with the following pattern: `Sprint_{TEAM_NAME}_{NUMBER}`. During the planning, a team selects issues from the backlog considering:
 - priority (high priority first)
 - area (default capability of the team first)
 - dependencies (unblock others)
 
-Contributors mark issues with the `WIP` label when they start working on it. When the work is done, the issue is closed.
+Contributors move the issue to "In progress" track in ZenHub when they start working on it. When the work is done, the issue is closed.
```
Comment (Disper) [inline issues-workflow.md]: I believe that it's more common to use "column" instead of "track".

Comment (sjanota) [reply issues-workflow.md]: In Zenhub it's called "pipeline", I'll change to that

Comment (kazydek) [inline issues-workflow.md]:
```suggestion
Contributors move the issue to the **In progress** column on the ZenHub board when they start working on it. When the work is done, the issue is closed.
```

Comment (kazydek) [reply issues-workflow.md]: I would also go for `column` - I know it is called pipeline on Zenhub but I don't think it is clear for sb who has not read ZenHub documentation ;)

**Comment by sjanota** [comment]
I had to recreate the branch on my fork. Here's new PR: https://github.com/kyma-project/community/pull/302

**Pull request closed without merging**